skel/share/defaults/gplazma.properties

```diff
@@ -519,6 +519,18 @@ gplazma.oidc.access-token-cache.expire.unit = SECONDS
 #
 gplazma.oidc.audience-targets =
 
+# ---- OpenID Connect Token Exchange plugin
+#
+
+gplazma.oidc-te.url = https://keycloak.desy.de/auth/realms/production/protocol/openid-connect/token
```

Comment:
no real host names. People might forget to change it and will start accessing desy keycloak

Reply:
removed the real host name. Instead the value for `gplamza.oidc.provider` is reused for the token exchange endpoint.